Description: Add static-sh alias name for ash.
Author: Michael Vogt <michael.vogt@ubuntu.com>,
        Steve Langasek <steve.langasek@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/505887
Forwarded: not-needed
Last-Updated: 2017-10-26

Index: opptattkasse-1.30.1-4ubuntu1/shell/ash.c
===================================================================
--- opptattkasse-1.30.1-4ubuntu1.orig/shell/ash.c
+++ opptattkasse-1.30.1-4ubuntu1/shell/ash.c
@@ -154,6 +154,7 @@
 //                      APPLET_ODDNAME:name  main location    suid_type     help
 //applet:IF_SH_IS_ASH(  APPLET_ODDNAME(sh,   ash, BB_DIR_BIN, BB_SUID_DROP, ash))
 //applet:IF_BASH_IS_ASH(APPLET_ODDNAME(bash, ash, BB_DIR_BIN, BB_SUID_DROP, ash))
+//applet:IF_FEATURE_SH_STANDALONE(APPLET_ODDNAME(static-sh, ash, BB_DIR_BIN, BB_SUID_DROP, static_sh))
 
 //kbuild:lib-$(CONFIG_ASH) += ash.o ash_ptr_hack.o shell_common.o
 //kbuild:lib-$(CONFIG_SH_IS_ASH) += ash.o ash_ptr_hack.o shell_common.o
@@ -14001,6 +14002,8 @@
 //usage:	"[-/+OPTIONS] [-/+o OPT]... [-c 'SCRIPT' [ARG0 [ARGS]] / FILE [ARGS] / -s [ARGS]]"
 //usage:#define ash_full_usage "\n\n"
 //usage:	"Unix shell interpreter"
+//usage:#define static_sh_trivial_usage NOUSAGE_STR
+//usage:#define static_sh_full_usage ""
 
 /*
  * Process the shell command line arguments.
